### ISSUE TITLE
Where is the source code used in the examples, please?

### ISSUE THREAD

**Issue by i-make-robots** [joel-costigliola/assertj-swing#263]
### What I found/tried:
I am new to AssertJ.  I'm reading https://joel-costigliola.github.io/assertj/assertj-swing-input.html 

There's a few lines of code pointing me in a good direction.  They assume I already know how to import everything needed.  The one minute quick-start doesn't have a runable demo.

I looked in various sub-folders for examples with no luck.

### What I expect:
A working example I can run locally to see these things in action for myself?  Same way a "how to use [swing feature x]" would have run-able source code.  Example, [JButton](https://docs.oracle.com/javase/tutorial/uiswing/components/button.html) links to [this demo](https://docs.oracle.com/javase/tutorial/displayCode.html?code=https://docs.oracle.com/javase/tutorial/uiswing/examples/components/ButtonDemoProject/src/components/ButtonDemo.java).

### separately: 
the quick-start code assumes I know the title of my frame.  As my app is multilingual I can't be sure what translation is running or the name of the frame around the panel I want to test.  Can application.start() return the root frame?

**Comment by i-make-robots** [comment]
Maybe my assumptions are way off.  i'm slowly realizing all classes that are testable in the demos create their own jframes in the default constructor?  is that right?  How Do I test a class that creates a single jpanel?

**Comment by scordio** [comment]
@i-make-robots have you tried looking at the Swing examples at https://github.com/assertj/assertj-examples?

**Comment by i-make-robots** [comment]
No.  I didn't mention them in "what I found/tried" in OP.  The link to those source files was not obvious to me.
